### PR TITLE
Feature/shortcut indication

### PR DESCRIPTION
**Pull request by dbuezas**
screen shot 2017-08-09 at 12 14 38

screen shot 2017-08-09 at 19 31 31

### CODE REVIEW

**Comment by dbuezas** [comment]
(builds upon Pull request 16)

**Comment by ssundarraj** [comment]
Hey, can you merge with master so I can just see the new diff?

Also, can you please add a link inline to dismiss the hints? It might be annoying for users to always have to see the hints.

**Comment by dbuezas** [comment]
Oh, I'll do the merging soon.
Regarding the link to the shortcut hints I think the images are not clear, time for a gif:
out
shortcut is always visible, and it changes to the hint when it is clicked.

I couldn't find a way to link directly to the chrome's extension keyboard shortcuts. It could be possible to set a shortcut directly from the extension itself through chrome.commands, but that would require a UI.

**Comment by dbuezas** [comment]
now it is done

**Comment by ssundarraj** [comment]
Sorry for the delay. I'll take a look next week.

File: src/css/main.css [hunk 1]
```diff
@@ -39,7 +39,25 @@ html{
 #highlighted {
 	background-color: #EEEEEE;
 }
+#shortcutInfo{
+	float: right;
+	margin: 3px;
+}
+#shortcutInfo a{
```
Comment (ssundarraj) [inline src/css/main.css]: nit: please add new lines after each rule

File: src/js/main.js [hunk 2]
```diff
@@ -183,6 +183,18 @@ async function initCommander() {
...
     document.onkeydown = handleKeydown;
+    const commands = await chromePromise.commands.getAll();
+    const mainCommand = commands.find(({name}) => name === '_execute_browser_action');
+
+    const shortcutEl = document.querySelector('#shortcutInfo #shortcutContainer a');
+    shortcutEl.innerText = mainCommand.shortcut;
+    shortcutEl.onclick = async function() {
+      document.querySelector('#shortcutInfo').classList.add('clicked');
+    };
+    document.querySelector('#shortcutInfo #guide a').onclick = async function() {
```
Comment (ssundarraj) [inline src/js/main.js:194]: Can you just make this an <a> tag?

Comment (dbuezas) [reply src/js/main.js:194]: sure, what do you mean with tag?

Comment (ssundarraj) [reply src/js/main.js:194]: Oops, I meant to write with an "a" tag.

i.e. `<a href="...">...</a>`

**Comment by ssundarraj** [comment]
> shortcut is always visible, and it changes to the hint when it is clicked.

Do you think an option to disable the shortcut it would be useful?

**Comment by ssundarraj** [comment]
@dbuezas Closing this due to inactivity.

**Comment by dbuezas** [comment]
Oh, sorry for that, I didn't even remember this PR.
In the meanwhile I made my own palette from scratch with modern tools https://github.com/dbuezas/chrome-palette/tree/master

**Pull request closed without merging**